### PR TITLE
Issue37 bolt api

### LINKED ISSUE #37: Update KarmaChameleon to use the new Slack Bolt API
https://api.slack.com/start/building/bolt-python

### PR DESCRIPTION
**Pull request by wrideout**
Fixes #37 

Refactors KarmaChameleon to use the latest Slack Bolt API!

### CODE REVIEW

**Comment by wrideout** [comment]
Also fixes #3

**Comment by wrideout** [comment]
I believe this also fixes #3

**Review by subecho — CHANGES_REQUESTED** [review]
This is a good start, but I'm concerned about the lack of modularity in the `karma_chameleon.py` file. With the effort that we took rewriting the thing with the Bolt API, we might want to also want clean that up as well.

File: README.md [hunk 1]
```diff
@@ -29,14 +31,49 @@ $ source .venv/bin/activate
...
+- chat:write.customize
+- chat:write.public
+- commands
+- users:read
+
+**User Token Scopes:**
+- channels:history
+
+There are two steps to running the bot on your local machine: running the actual Python code and
+forwarding the port that the Flask server is running on to a publicly accessible URL. I used
```
[the remark this answers is not in the document]
Comment (wrideout) [reply README.md]: No good reason other than I wasn't sure how that design would work with the API.  I will make the change.

File: karma_chameleon.py [hunk 2]
```diff
@@ -0,0 +1,273 @@
+# Karma Chameleon
```
[the remark this answers is not in the document]
Comment (wrideout) [reply karma_chameleon.py:1]: This has been reflectored.

File: karma_chameleon.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+    Arguments:
+    message -- dictionary representation of the message which contains a karma operation
+    say -- method for printing back to the same channel from which the command was run
+    """
+    rsp = app.decrement_karma(message)
+    say(rsp)
+
+
+@app.command("/ec_leaderboard")
```
Comment (wrideout) [inline karma_chameleon.py:120]: This needs to be put back... The command present now is debug I forgot to remove.

**Review by subecho — APPROVED** [review]

**Pull request merged**